patterns/1-initial/innersource-hackathon.md

```diff
@@ -0,0 +1,80 @@
+
+## Solution
+
+Organize a company wide hackathon focused on InnerSource. An InnerSource hackathon provides a safe space for engineers to try and contribute to InnerSource projects without any presumption and prejudice. It could be a 1 or 2 day event.
+
+It can preferrable be organized by InnerSource Program Office (ISPO), if it exists in the organization or by the Open Source Program Office (OSPO), if OSPO also drives InnerSource within the company. It can also be organized by other common service organizations too, if need be. Basically a central team or a group of individuals, who believe in InnerSource, can organize the hackathon.
+
+All engineers in the organization can participate in the hackathon. The participants can be new to InnerSource, or InnerSource practitioners already. They can participate individually or as a team. Participating as a team also provides a safe environment, for example, those are new to InnerSource can team up InnerSource practitioners.
+
+There could be one or more categories to participate as follows:
+
+* Start a new InnerSource project: It could be either starting a new project from scratch as InnerSource or making a existing project InnerSource ready.
+* Contribute to the existing InnerSource project: InnerSource project owners can list down the features in GitHub issues as a pre-requisite before the hackathon and the participants can contribute to it during the hackathon.
+* Participate as an InnerSource mentor: Participants can also mentor new InnerSource practitioners and provide knowledge transfer or coaching during the event.
+
+The event could be held virtually or physically in different locations in the organization based on the cost and budget involved. A virtual event will have best ROI but it could depend on each organization.
+
+The winners and participants should be recognized and acknowledged in a company wide forum at the end of the hackathon. This is important as it keeps motivating them and more engineers to adopt and practice InnerSource going forward.
+
+Such an event provides a safe space for the engineers who want to adopt or contribute to InnerSource but didn't have the time and motivation to do it or for those who kept deprioritizing due to high priority work deliverables. From middle management point of view, 1 or 2 days for such an event is not much of an ask and hence they are more likely to accept.
+
+## Resulting Context
+
+* There is increased adoption of InnerSource by providing such a safe platform to try and experiment combined with recognition and support by the senior leadership and middle management.
+* It results in more projects being made InnerSource ready and innersourced, and also results in more InnerSource contributions and start building community around existing InnerSource projects.
```

Comment:
A project can be InnerSource ready if it has minimum pre-requisites met. For example:
- if a repository has [standard base documentation](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/base-documentation.md) (and/or)
- if the corresponding project maintainers have agreed to innersource the project and set aside time for addressing queries or contributions from the community (and/or) 
- if there is a proper [communication tooling](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/communication-tooling.md) for the project, that the community can use to reach the maintainers easily. 

By innersourced, I mean that the project is announced as an InnerSource project within the company and officially made open for contributions. It can also include listing the project in the [InnerSource Portal](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/main/patterns/2-structured/innersource-portal.md) if used by the company. 

If a project is innersourced, it means that the repository and the team have already met InnerSource-ready criteria.

However, these criteria and definitions can differ in each company, but this is what I meant by 'make InnerSource ready' and 'innersourced' :)

Reply:
Got it. Typically we try not to use the verb form of "innersourced".

Let me try to make a proposal, that would also split up this line into more distinct benefits.

```suggestion
* More InnerSource projects are published within the company.
* The InnerSource projects receive more contributions.
* Communities start to form around these InnerSource projects.
```